_sass/imports/_animations.scss

```diff
@@ -1,16 +0,0 @@
-@-webkit-keyframes fadeIn20 {
```

Comment:
These animation settings are used by `_helpers.scss` (116 `.bg-img`), so that reference should also be removed if this file is deleted.